frontend/ui/translator.lua

```diff
@@ -232,11 +236,33 @@ function Translator:genSettingsMenu()
                     local lang = G_reader_settings:readSetting("translator_from_language")
                     return T(_("Translate from: %1"), self:getLanguageName(lang, ""))
                 end,
+                help_text = _("Selecting a specific source language will have it used with all text on all your books."),
```

Comment:
```suggestion
                help_text = _("If a specific source language is manually selected, it will be used everywhere, in all your books."),
```

(I initially went with something along the lines of "make this choice stick across books", but that might be harder to grok for TLs).